_posts/newsletters/2024-05-28-newsletter.md

```diff
@@ -0,0 +1,349 @@
+
+### **💡Hamlyn Symposium in London: Open-Source Software for Surgical Technologies**
+
+Open-source software libraries for surgical technologies include areas on computer assisted surgery, medical robotics and medical imaging (e.g. SciKit-Surgery, 3DSlicer, Holohub, MONAI, PLUS, KitwareMedical, and FAST). These libraries have shown great progress in the last twenty years due to their rapid innovation and adaption to new technologies, continuous release of open software and datasets for algorithm evaluation, good documentation and educational resources for students, researchers, engineers and clinicians, and partnerships between academia and industry. 
+
+Details: [https://www.hamlynsymposium.org/events/open-source-software-for-surgical-technologies-2/](https://www.hamlynsymposium.org/events/open-source-software-for-surgical-technologies-2/)
+
+
+
+### Running List of Conferences and Events 
+
+[Previously shared conferences and events.](https://docs.google.com/document/d/1l6zf3oHlKP3dLapR6QxQMMAD1c76-4UlR1WEKM4YVOk/edit?usp=sharing)
+
+-----------------
+
+<a name="reads"></a>
+# **6. Interesting Reads, Videos, or Podcasts**
+* [Code for Thought: Changing the Landscape - The Alan Turing Skills Policy Award - Part II](https://codeforthought.buzzsprout.com/1326658/15073663-en-changing-the-landscape-the-alan-turing-skills-policy-award-part-ii) by Peter Schmidt
+* [Code for Thought: Carpentries for All - Greg Wilson - Code for Thought](https://codeforthought.buzzsprout.com/1326658/15010367-en-carpentries-for-all-greg-wilson) by Peter Schmidt
+* [Code for Thought: Collaborations Workshop 2024](https://codeforthought.buzzsprout.com/1326658/15045013-en-collaborations-workshop-2024) by Peter Schmidt
+* [Connecting research computing, data & software professionals](https://coco.cyberinfrastructure.org/) by Community of Communities
+*  [Eleven strategies for making reproducible research and open science training the norm at research institutions](https://elifesciences.org/articles/89736) by eLife
+*  [Ten simple rules for pushing boundaries of inclusion at academic events](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011797) by Siobhan Mackenzie Hall, Daniel Kochin, Carmel Carne, Patricia Herterich, Kristen Lenay Lewers, Mohamed Abdelhack, Arun Ramasubramanian, Juno Felecia, Michael Alphonse, Visotheary Ung, Sara El-Gebali, Christopher Brian Currin, Esther Plomp, Rachel Thompson, Malvika Sharan  
+*  [Inclusive Language in Technology](https://www.aswf.io/blog/inclusive-language/) by Academy Software Foundation
+*  [Racial Equity Tools Glossary](https://www.racialequitytools.org/glossary) by Racial Equity Tools
+*  [There’s no such thing as sustainable research software](https://danielskatzblog.wordpress.com/2024/05/13/no-sustainable-research-software/) by Daniel S. Katz
+
+-----------------
+
+<a name="financials"></a>
+# **7. US-RSE Financials**
```

Comment:
Is this exactly the same as last month? If there is nothing new, I'd suggest deleting the whole section. 
I realize this level of content comment is beyond the normal website review, so feel free to ignore this time.

Reply:
By "this" I mean the entire Financials section.